### PR TITLE
Feature/applying chai spies

### LINKED ISSUE #29: Apply Chai Spies
As a developer, I would like to know that jquery is being called on the dom.

### PR DESCRIPTION
**Pull request by okalimbet**
## What is the change? 
Refactoring script.js by creating a new file dom-display.js. Dom manipulating functions were moved from script.js to dom-display.js Chai-spies testing was added for api-request.js and dom-display.js

Small fixes in Activity, User-repo and userrepo-tests files.

## What does it fix? 
Syntax

## Is this a fix or a feature? 
Feature/small fixes

## Where should the reviewer start? 
Activity.js, User-repo.js - syntax fixes

dom-display.js:
line 1 - creating a domDisplay object literal

script.js:
line 19-end - dom-display was imported, functions that are related to dom manipulation were moved.

apiRequest-test.js:
line 1 - test for api requests

dom-displya-test.js:
line 1 - tests for dom-display.js

userrepo-test.js:
line 59 - removed console.log

## How should this be tested?
By running npm test should which should show all successfully passed tests. By running npm start the browser should display all random user related information.

### CODE REVIEW

File: src/dom-display.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import $ from 'jquery';
+
+let domDisplay = {
```
Comment (cameronRomo) [inline src/dom-display.js:3]: Very nice separation of the dom display methods

File: test/dom-display-test.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      'makeStepsHTML',
+      'makeStairsHTML',
+      'makeMinutesHTML',
+      'addFriendGameInfo',
+      'makeFriendChallengeHTML',
+      'makeStepStreakHTML'
+    ], () => {});
+  });
+
+  after(function() {
```
Comment (cameronRomo) [inline test/dom-display-test.js:44]: I like this after hook to clear out the domDisplay

**Review by cameronRomo — APPROVED** [review]
Very nice use of chai spies. Seems like a good practice to use moving forward!

File: test/apiRequest-test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+chai.use(spies);
+
+const expect = chai.expect;
+
+
+describe('apiRequest', function() {
+
+  before(function() {
+    global.apiRequest = {};
```
Comment (mdflynn) [inline test/apiRequest-test.js:12]: Nice work using spies!

**Review by mdflynn — APPROVED** [review]
Nice work separating the dom methods and testing with spies

**Pull request merged**